docs/testing/stores.md

```diff
@@ -113,7 +130,14 @@ describe('PetStore', () => {
   // lets use this inaccessible method to show how we can test
   // non static methods if we desire/need to
   it('rounds money to 2 decimal places', function() {
-    var unwrappedStore = new UnwrappedPetStore();
+    class TestPetStore extends UnwrappedPetStore {
+      constructor() {
+        // we extend and override the constructor because
```

Comment:
Actually I am going to go ahead and mock `bindActions`. Though most testing might be able to be achieved through blackbox testing, I want to provide examples for users that wish to test the class directly or have a use case where blackbox testing is not enough.